core/trino-main/src/main/java/io/trino/sql/planner/iterative/rule/AggregationDecorrelation.java

```diff
@@ -32,10 +32,10 @@ private AggregationDecorrelation() {}
 
     public static boolean isDistinctOperator(PlanNode node)
     {
-        return node instanceof AggregationNode &&
-                ((AggregationNode) node).getAggregations().isEmpty() &&
-                ((AggregationNode) node).getGroupingSetCount() == 1 &&
-                ((AggregationNode) node).hasNonEmptyGroupingSet();
+        return node instanceof AggregationNode an &&
```

Comment:
Can the tool be configured to avoid abbreviations like this?

Reply:
No idea tbh.

Reply:
It's using these abbreviations only when there is a collision of the default name in the scope I think